packages/cli/src/__tests__/ceramic-multi-daemon.test.ts

```diff
@@ -16,16 +16,20 @@ const seed = 'SEED'
 const TOPIC = '/ceramic'
 
 const makeCeramicCore = async (ipfs: IpfsApi, stateStoreDirectory: string): Promise<Ceramic> => {
-  const core = await Ceramic.create(ipfs, {
-    pubsubTopic: TOPIC,
-    stateStoreDirectory,
-    anchorOnRequest: false,
-    indexing: {
-      db: `sqlite://${stateStoreDirectory}/ceramic.sqlite`,
-      disableComposedb: false,
+  const core = await Ceramic.create(
+    ipfs,
+    {
+      pubsubTopic: TOPIC,
+      stateStoreDirectory,
+      anchorOnRequest: false,
+      indexing: {
+        db: `sqlite://${stateStoreDirectory}/ceramic.sqlite`,
```

Comment:
is sqlite hardcoded in?  just a side question

Reply:
This is a test file, so yes this test always uses sqlite